feat(MathJax): Add support for MathJax extension, e.g. physics package

huanyushi opened the pull request:
## Type of change
<!-- Please select the desired item checkbox and change it from `[ ]` to `[x]` and then delete the irrelevant options. -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Improvement (refactoring and improving code)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Description
<!--
  Please include a summary of the change and which issue is fixed. 
  Please also include relevant motivation and context. 
  List any dependencies that are required for this change.
-->
MathJax provides many extensions that help us input mathematical expressions more conveniently. Among them, the [physics package](https://docs.mathjax.org/en/latest/input/tex/extensions/physics.html) is particularly important for STEM majors, as it allows us to easily input matrices, Dirac notation, adjust bracket sizes, and more.

I modified `_includes/js-selector.html` to add support for the `physics` package. I also added an example using the `physics` package in `_posts/2019-08-08-text-and-typography.md`.

More extensions of MathJax can be found [here](https://docs.mathjax.org/en/latest/input/tex/extensions/index.html)

## Additional context
<!-- e.g. Fixes #(issue) -->
Fixed #622

cotes2020 reviewed (commented):
Thanks @huanyushi, this PR just loads a certain extension (`physics`).  I think it would be possible to move the mathjax configuration to a js config file so that extensions can be configured on demand.

huanyushi commented:
> I think it would be possible to move the mathjax configuration to a js config file so that extensions can be configured on demand.

@cotes2020 Of coure, I have updated this PR. I moved the MathJax configuration from `_includes/js-selector.html` to a new file `_includes/mathjax-support.html`.

I removed the `physics` extension that I had added. Users can now add any configurations they need to `_includes/mathjax-support.html`.

cotes2020 commented:
See #1670
